src/Core/Mesh/TopologicalTriMesh/TopologicalMesh.cpp

```diff
@@ -27,7 +27,7 @@ void addAttribPairToTopo( const TriangleMesh& triMesh, TopologicalMesh* topoMesh
 }
 
 template <typename T>
-void addAttribPairToCore( TriangleMesh& triMesh, TopologicalMesh* topoMesh,
+void addAttribPairToCore( TriangleMesh& triMesh, const TopologicalMesh* topoMesh,
                           OpenMesh::HPropHandleT<T> oh, std::vector<PropPair<T>>& vprop ) {
     AttribHandle<T> h{triMesh.addAttrib<T>( topoMesh->property( oh ).name() )};
```

Comment:
why not keeping code consistent with `Attr h = ... ` ?

Reply:
also, why not using `const Handle& h` in argument? (same for functions below)

Reply:
Well, I'm not sure to catch the first comment.
For the second, as a PropHandle is something like an int, and all OpenMesh method take arg by value, I don't think we need to use const ref.